lib/YuiRestClient.pm

```diff
@@ -89,6 +90,17 @@ sub setup_libyui_running_system {
     set_app($app);
 }
 
+sub setup_libyui_firstboot {
+    my $port = get_var('YUI_PORT');
+    add_suseconnect_product('sle-module-development-tools') if is_sle;
```

Comment:
we should not introduce 'is_sle', I put in previous method 'add_suseconnect_product' in previous function because there is not yet any job in the running system for openSUSE but it should be decoupled.

Reply:
This is the same question as addressed by @sofiasyria, I don't really know how to address it without doing even worst, like having two different functions and modules. some test data maybe ? Otherwise we can do this temporarily as we plan to make a specific hdd image with libyui already installed.